### PR TITLE
Added test data with instructions

### PR DESCRIPTION
**Pull request by andykitch**

### CODE REVIEW

File: src/API/src/schema.gql [hunk 1]
```diff
@@ -45,6 +46,21 @@ type Query {
   userRole(id: String!): UserRole!
 }
 
+"""sample data"""
+type Sample {
```
Comment (andykitch) [inline src/API/src/schema.gql:50]: Not sure why this was added - maybe I ran a build that hadn't been run before? Either way, we want it here anyway...

File: docs/SMG/10-working-with-database.md [hunk 2]
```diff
@@ -13,7 +13,9 @@ This will create a `migrations` table in the db, storing a list of all migration
 
 ## Data load
 There currently exists two endpoints on the API for data ingest:
```
Comment (ray-millward-tessella) [inline docs/SMG/10-working-with-database.md:15]: Can you give instructions on how to call the endpoint, I've briefly seen Chris and Colin try this and they have to set some specifics about form-data etc.

Comment (andykitch) [reply docs/SMG/10-working-with-database.md:15]: Yep, done

**Review by ray-millward-tessella — APPROVED** [review]

File: src/API/src/schema.gql (hunk 1 above)
Comment (ray-millward-tessella) [reply src/API/src/schema.gql:50]: We should try to work this out and make sure it isn't some sort of configuration specific to your machine. Not something for this PR though.

**Pull request merged**